modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -4,4 +4,24 @@ If you encounter errors when attempting to run Anypoint CLI commands, troublesho
 
 include::api-governance::partial$cli-commands-troubleshoot.adoc[tag=before]
 
-include::api-governance::partial$cli-commands-troubleshoot.adoc[tag=errors]
\ No newline at end of file
+include::api-governance::partial$cli-commands-troubleshoot.adoc[tag=errors]
+
+== Windows Command Syntax
+
+Some commands that include special characters like backslashes `\` or pipes `|` do not behave as expected on Windows systems because of how the Windows Command Prompt interprets these characters. +
```

Comment:
```suggestion
Some commands that include special characters like backslashes `\` or pipes `|` don't behave as expected on Windows systems because Windows Command Prompt interprets these characters in a different way. +
```